plugins/h3/lib_h3.ecllib

```diff
@@ -0,0 +1,52 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the License);
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an AS IS BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+EXPORT h3_index_t := UNSIGNED8;
+EXPORT h3_stringIndex_t := VARSTRING;
+EXPORT h3_degrees_t := REAL8;
+EXPORT h3_rads_t := REAL8;
+EXPORT h3_resolution_t := UNSIGNED4;
+EXPORT h3_latlng_t := SET OF h3_degrees_t;
+EXPORT h3_boundary_t := SET OF h3_degrees_t;
+
+EXPORT h3 := SERVICE : plugin('h3'), library('h3'), namespace('h3'), CPP, PURE
+
+  //  Indexing  ---
+  h3_index_t index(CONST h3_degrees_t lat, CONST h3_degrees_t lng, CONST h3_resolution_t resolution) : cpp,action,context,fold,entrypoint='index';
```

Comment:
Does "CONST" mean anything for numerics?

Reply:
I don't think so.